README.md

```diff
@@ -1,6 +1,15 @@
 # flotta-scale-tests
 The purpose of this project is to test the scalability and performance of [project-flotta](https://github.com/project-flotta/flotta-operator) using [JMeter](https://jmeter.apache.org).
 
+## Install OCP SNO cluster
+Clone https://gitlab.cee.redhat.com/jgil/simple-openshift-cluster-provisioner
+Run `CLUSTER_NAME=<cluster name> SNO_MASTER_MEMORY=<amount of memory in Mib> SNO_MASTER_CPU=<amount of CPU> make create_sno_cluster`.
+
+Ex: `CLUSTER_NAME=devicemgmt1 SNO_MASTER_MEMORY=64000 SNO_MASTER_CPU=16 make create_sno_cluster`
+## Configure K8S/OCP cluster
+Follow https://metallb.universe.tf/installation/ to avoid having to create NodePort service to expose the flotta operator
+See https://metallb.universe.tf/configuration/#layer-2-configuration. Use `kubectl get nodes -o` or `oc get nodes -o wide` to get the IP adress.
```

Comment:
the first command is incomplete:
```
kubectl get nodes -o
```

Reply:
I deleted this part of the README to use kubectl port-forward and now NodePort service so that part is no longer relevant, maybe it will be if we see any limitations with the NodePort and that we want to use LoadBalancer instead